Common/Product/SharedProject/CommonUtils.cs

```diff
@@ -200,6 +200,26 @@ public static string GetAbsoluteFilePath(string root, string relativePath)
             return string.Join(Path.DirectorySeparatorChar.ToString(), segments);
         }
 
+        /// <summary>
+        /// Returns a normalized file path created by joining relativePath to root.
+        /// The result is not guaranteed to end with a backslash.
+        /// </summary>
+        /// <returns>True, if the absolut path is returned successfully; otherwise false, 
+        /// if root is not an absolute path, or either path is invalid.</returns>
+        public static bool TryGetAbsoluteFilePath(string root, string relativePath, out string absoluteFilePath)
```

Comment:
>TryGetAbsoluteFilePath [](start = 27, length = 22)

This feels specialized for the only caller, so I'd consider inlining it.

Reply:
Although I agree is only used for this scenario, I feel like adding it to this class is a good practice to as it shares the same responsibility than the rest of the methods on it as well as making the code readable and cleaner. 

If you still have preference for inlining it, I'm happy to change it.

Reply:
Changed as per our offline comments.